docs/send-data/kubernetes/install-helm-chart.md

```diff
@@ -24,20 +24,23 @@ helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collect
 helm repo update
 ```
 
-### Sumo Logic Account
+### Sumo Logic account
 
-If you don’t already have a Sumo account, you can create one by clicking the Free Trial button on https://www.sumologic.com/.
+If you don’t already have a Sumo Logic account, you can [Start a free trial](https://www.sumologic.com/sign-up).
 
-The following are required to set up Sumo Logic's Kubernetes collection.
+The following are required to set up and deploy the Sumo Logic Kubernetes collection.
 
-- An [Access ID and Access Key](/docs/manage/security/access-keys/) with
-  [Manage Collectors](/docs/manage/users-roles/roles/role-capabilities#data-management) capability.
-- Please review our [minimum requirements](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/README.md#minimum-requirements) and [support matrix](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/README.md#support-matrix)
+- Review our [minimum requirements](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/README.md#minimum-requirements) and [support matrix](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/README.md#support-matrix)
+- An [Access ID and Access Key](/docs/manage/security/access-keys/) with the following [Data Management role capabilities](/docs/manage/users-roles/roles/role-capabilities#data-management):
+   - Manage Collectors
+   - Manage Fields
+   - Manage Content
+   - Manage Apps
+   - Manage Monitors
+   - ...you need View capabilities on most other non-admin/non-security pieces as well.
```

Comment:
To be specific, `Manage Collectors` and `Manage Fields` are the only strictly necessary capabilities. Selecting these will also select the necessary View capabilities, they don't need to be selected separately.

`Manage Monitors` and `Manage Content` are necessary to install the built-in monitors and dashboards, which are recommended and enabled by default. But if the capabilities aren't present, the content will simply not be installed, and the Helm Chart will otherwise be successfully installed.

`Manage Apps` is not necessary.

Reply:
@dkarabin-sumo ☝🏻

Reply:
fixed!